cypress/e2e/tests/language-switcher.cy.js

```diff
@@ -1,124 +1,24 @@
-    //     cy.get(options[0]).find('li').then(li => {
-    //       cy.get(li[5]).click()
-    //       cy.get('[data-test="vote-logo"]').should('contain', 'Vote.gov हिन्दी में')
-    //     })
-    //     )
-    // })
-    //
-    // it(`test khmer ${page}`, () => {
-    //   cy.visit(page)
-    //   cy.get('[data-test="language-button"]').click().get('[data-test="language-switcher"]').then(options =>
-    //     cy.get(options[0]).find('li').then(li => {
-    //       cy.get(li[6]).click()
-    //       cy.get('[data-test="vote-logo"]').should('contain', 'Vote.gov ជាភាសាខ្មែរ')
-    //     })
-    //     )
-    // })
-    //
-    // it(`test korean ${page}`, () => {
-    //   cy.visit(page)
-    //   cy.get('[data-test="language-button"]').click().get('[data-test="language-switcher"]').then(options =>
-    //     cy.get(options[0]).find('li').then(li => {
-    //       cy.get(li[7]).click()
-    //       cy.get('[data-test="vote-logo"]').should('contain', 'Vote.gov 한국어 ')
-    //     })
-    //     )
-    // })
+      languagesTranslations.forEach((languageTranslation) => {
 
-    // it(`test navajo ${page}`, () => {
-    //   cy.visit(page)
-    //   cy.get('[data-test="language-button"]').click().get('[data-test="language-switcher"]').then(options =>
-    //     cy.get(options[0]).find('li').then(li => {
-    //       cy.get(li[8]).click()
-    //       cy.get('[data-test="vote-logo"]').should('contain', 'Vote.gov Diné')
-    //     })
-    //     )
-    // })
-  //
-  //   it(`test tagalog ${page}`, () => {
-  //     cy.visit(page)
-  //     cy.get('[data-test="language-button"]').click().get('[data-test="language-switcher"]').then(options =>
-  //       cy.get(options[0]).find('li').then(li => {
-  //         cy.get(li[9]).click()
-  //         cy.get('[data-test="vote-logo"]').should('contain', 'Vote.gov sa Tagalog')
-  //       })
-  //       )
-  //   })
-  //
-  //   it(`test vietnamese ${page}`, () => {
-  //     cy.visit(page)
-  //     cy.get('[data-test="language-button"]').click().get('[data-test="language-switcher"]').then(options =>
-  //       cy.get(options[0]).find('li').then(li => {
-  //         cy.get(li[10]).click()
-  //         cy.get('[data-test="vote-logo"]').should('contain', 'Vote.gov bằng Tiếng Việt')
-  //       })
-  //       )
-  //   })
-  //
-  // it(`test yup'ik ${page}`, () => {
-  //   cy.visit(page)
-  //   cy.get('[data-test="language-button"]').click().get('[data-test="language-switcher"]').then(options =>
-  //     cy.get(options[0]).find('li').then(li => {
-  //       cy.get(li[11]).click()
-  //       cy.get('[data-test="vote-logo"]').should('contain', 'Vote.gov Akuzipigestun')
-  //     })
-  //     )
-  // })
-  })
-})
+        cy.get('[data-test="language-button"]').click();
+
+        cy.contains(languageTranslation.language).click({ force: true });
+
+        cy.get(".logo-text").should("contain.text", languageTranslation.logoText);
```

Comment:
Can you update the `".log-text"` to grab `data-test="vote-logo"` to keep it consistent with others and to make sure if the dev changes names we are still good.